terraform/ecs/efs.tf

```diff
@@ -84,6 +84,11 @@ resource "aws_instance" "collector_efs_ec2" {
     Name = "Integ-test-aoc"
   }
 
+  metadata_options {
+    http_endpoint = "enabled"
+    http_tokens   = "required"
+  }
+
```

Comment:
Has this change been tested on the dev branch? Are ecs tests ran on the dev branch?

Reply:
ecs tests are not ran on the dev branch and so I will have to do more tests locally for these instances.  For now, I will remove the change for the ecs tests and merge the PR to verify that the changes work for the ec2 instances first.